--- src/cy16cc/cy16_codegen.c
+++ src/cy16cc/cy16_codegen.c
@@ -216,7 +216,9 @@
         fprintf(output, ".global _%s\n", fn->name);
         fprintf(output, "_%s:\n", fn->name);
         if (fn->init_data) {
-          error("initialized globals not fully supported in simple codegen");
+          for (int i = 0; i < fn->ty->size; i++) {
+             fprintf(output, "    .byte %d\n", fn->init_data[i]);
+          }
         } else {
           fprintf(output, "    .word 0\n"); // Just a placeholder word
         }
